Sources/Mapper.swift

```diff
@@ -80,19 +82,25 @@ public struct Mapper {
      - parameter key: The key to retrieve from the source data, can be an empty string to return the entire
```

Comment:
Note: parameter name is outdated. Would be nice to change all instances of `key` to `field` here.